sql/2020/11_Security/hsts_attributes.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# HSTS includeSubDomains and preload usage
+SELECT
+  client,
+  COUNT(0) AS freq,
+  COUNTIF(REGEXP_CONTAINS(hsts_header_val, r'max-age\s*=\s*[^0]')) / COUNT(0) AS pct_valid_max_age,
```

Comment:
Missed a couple of insensitive matches here so best to double check all the `REGEX_` functions.

```suggestion
  COUNTIF(REGEXP_CONTAINS(hsts_header_val, r'(?i)max-age\s*=\s*[^0]')) / COUNT(0) AS pct_valid_max_age,
```

Thoughout